src/dmd/expressionsem.d

```diff
@@ -7076,8 +7076,8 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
                 auto tFrom = t1b.nextOf();
                 auto tTo = tob.nextOf();
 
-                // https://issues.dlang.org/show_bug.cgi?id=19954
-                if (exp.e1.op != TOK.string_ || tTo.ty == Tarray)
+                // https://issues.dlang.org/show_bug.cgi?id=20130
+                if (exp.e1.op != TOK.string_ || !ex.isStringExp)
```

Comment:
That expression is always `true`, see https://github.com/dlang/dmd/blob/1499a0d4bde7b2a6909cc31ef1108d67c54e86ae/src/dmd/expression.d#L1527

Reply:
No, exp.e1 vs ex.

Reply:
Nevermind